testdata/baselines/reference/submodule/conformance/jsDeclarationsTypeReferences2.symbols.diff

```diff
@@ -1,17 +1,14 @@
 --- old.jsDeclarationsTypeReferences2.symbols
 +++ new.jsDeclarationsTypeReferences2.symbols
-@@= skipped -3, +3 lines =@@
- const{ a, m } = require("./something").o;
- >a : Symbol(a, Decl(index.js, 0, 6))
+@@= skipped -5, +5 lines =@@
  >m : Symbol(m, Decl(index.js, 0, 9))
-->require("./something").o : Symbol(o, Decl(something.ts, 0, 12))
-->require : Symbol(require)
+ >require("./something").o : Symbol(o, Decl(something.ts, 0, 12))
+ >require : Symbol(require)
 ->"./something" : Symbol("something", Decl(something.ts, 0, 0))
```

Comment:
Still missing _something_ here, as this is mad about the actual string not having resolved to the other module (which would break something like go to def).

Reply:
It's probably still missing because of `getSymbolAtLocation` doesn't support js declaration lookups yet (since it's bound entirely differently), or something similar.